docs/concepts/executor/serve.md

```diff
@@ -157,6 +155,33 @@ The above example deploys the `DummyHubExecutor` from Executor Hub into your Kub
 The Executor you use needs to be already containerized and stored in a registry accessible from your Kubernetes cluster. We recommend Executor Hub for this.
 ````
 
+Once the Executor is deployed, you can expose a service:
```

Comment:
What's the reason behind not exposing the service automatically?

Reply:
Nevermind. This is only for the external IP.

Reply:
I guess it depends on how the user wants to do so, which varies from kubernetes provider to another.
 I guess the service we expose already is mostly useful for jcloud. Not sure how a user would leverage it without portforwarding